api/Window.json

```diff
@@ -9092,15 +9094,16 @@
             },
             "firefox": [
               {
-                "version_added": true
+                "version_added": "20"
               },
               {
-                "version_added": "20",
+                "version_added": "1",
+                "version_removed": "20",
                 "notes": "The mimimal size is clamped to prevent the window from being too small for the user to interact with."
```

Comment:
Doesn't the note belong with the newest entry? I tried to figure out if the clamping is still there in https://github.com/mozilla/gecko-dev/blob/168c45a7acc44e9904cfd4eebcb9eb080e05699c/dom/base/nsGlobalWindowOuter.cpp#L5608 but am not sure.

Getting this note right doesn't seem all that important since it was so far in the past. With the blessing of @ddbeck perhaps we can just remove it?

Reply:
This could folded into one entry with a note beginning "Before version 20, the minimal size is…".

That is, if it's truly old behavior.

Reply:
Apologies, forgot to move the note when updating this data!  I've flattened it down into one support statement.